### PR TITLE
#375 Task instances link from template view page

### PR DESCRIPTION
**Pull request by vimala-nagarja-CTL**
### Brach name should have following pattern
issue_\<number\>_githubId  for example pull request related to 81 by Manoj will be named as issue_81_magrawa
### Once you commit your changes to new branch you can follow instructions here for creating a pull request
[Create a New Pull Request](https://help.github.com/articles/creating-a-pull-request/)

[Learn more about pull requests](https://help.github.com/articles/about-pull-requests/)

### CODE REVIEW

**Comment by vimala-nagarja-CTL** [comment]
Hi Don,
  Kindly review the changes.
Thanks,
Vimala G.N

File: mdw-hub/web/js/assets.js [hunk 1]
```diff
@@ -294,15 +294,17 @@ assetMod.controller('PackageController', ['$scope', '$routeParams', '$route', '$
   };
 }]);
 
-assetMod.controller('AssetController', ['$scope', '$routeParams', '$location', 'mdw', 'util', 'uiUtil', 'Assets', 'Asset', 
-                                       function($scope, $routeParams, $location, mdw, util, uiUtil, Assets, Asset) {
+assetMod.controller('AssetController', ['$scope','$cookieStore', '$routeParams', '$location', 'mdw', 'util', 'uiUtil', 'Assets', 'Asset',
```
Comment (donaldoakes) [inline mdw-hub/web/js/assets.js]: formatting: space between function params

File: mdw-hub/web/assets/asset.html [hunk 2]
```diff
@@ -32,6 +32,9 @@
           </a>            
           <a href="" class="btn btn-default" ng-if="authUser.hasRole('Site Admin')" ng-click="deleteAsset()">
             <span class="glyphicon glyphicon-remove"></span> Delete
+          </a>  
+          <a ng-if="task" href="" class="btn btn-default" ng-if="authUser.hasRole('Site Admin')" ng-click="viewInstances()">
```
Comment (donaldoakes) [inline mdw-hub/web/assets/asset.html]: **Site Admin** is not required to view task instances.  You don't have to check any particular role at this time.

**Comment by vimala-nagarja-CTL** [comment]
Hi Don,
  I incorporated review comments.
Thanks,
Vimala G.N

**Pull request merged**